README.md

```diff
@@ -32,19 +31,19 @@ The bug seems to exist on 2.X sound app too but the rop gadgets and the va to pa
 | stage2 payload constants | ✓ | ✗ | ✗ |
 
 ## Installation
-1. Download the relevant soundhax-region-console.m4a file for your device.
+1. Download the relevant soundhax-region-console-firmware.m4a file for your device.
 2. Save the soundhax song file and copy to the root of your SD.
 3. Download the [otherapp payload](https://smealum.github.io/3ds/) for your 3DS version, rename it to `otherapp.bin`, and copy it to the root of the SD card.
-4. Download the [Homebrew Starter Kit](https://smealum.github.io/ninjhax2/starter.zip) and unzip to the root of the SD card (if it is not there already).
+4. Download the [Homebrew Menu](https://github.com/fincs/new-hbmenu/releases/latest) and place `boot.3dsx` in the root of the SD card (if it is not there already).
```

Comment:
What's the situation here with Homebrew Menu exactly? I only see one release for this and I wonder who is actively maintaining it now. It's fincs?

Reply:
>  I only see one release for this

Ah, that's because I've linked to releases/latest. I can change it if you want.

>  I wonder who is actively maintaining it now. It's fincs?

Yes it's @fincs . I also have write access to it, just in case.

Reply:
The old homebrew menu was "discontinued" years ago, back when Rosalina 3dsx loading support was introduced. The most recent working version of a 3DS homebrew menu is the one this PR changes the link to point to (which is a complete rewrite of the old one). Unfortunately the "ninjhax2 starter kit" was never updated to reflect this, so linking to it is misleading nowadays.

Reply:
Sounds good, thanks for clarifying.